src/Uno.Templates/content/unoapp/.template.config/template.json

```diff
@@ -16,7 +16,7 @@
   "name": "Uno Platform App",
   "identity": "Uno.Platform.UnoApp.WinUI.netcoremobile.CSharp",
   "groupIdentity": "Uno.Platform.UnoApp.WinUI.netcoremobile",
-  "description": "Solution template for creating a multi-platform C# & XAML application targeting Windows App SDK, Android, iOS, macOS, WebAssembly, Skia Desktop.",
+  "description": "Solution template for creating a multi-platform C# & XAML application targeting Android, iOS, WebAssembly, Skia Desktop Windows, macOS, Linux (X11/Framebuffer) and Windows App SDK.",
```

Comment:
@jeromelaban, @kazo0 to revise if you think this updated version makes sense, but to me it matches the order and platform details in the "Platforms" section of the wizard

Reply:
Maybe "Skia Desktops" instead of listing out all of the desktop targets

Reply:
Jerome is okay with the current version